### PR TITLE
added a new processor for finding isolated photon

### PR DESCRIPTION
**Pull request by tjunping**
BEGINRELEASENOTES
- added a new processor for finding isolated photon
- the default option for isolated muon finder is changed back to use Yoke energy

ENDRELEASENOTES

### CODE REVIEW

**Comment by rete** [comment]
@gaede @andresailer @petricm Anyone knows what's happening with Travis-CI ? Nothing is sent their servers. This is the case for all repositories in iLCSoft. Looks fine for AIDASoft for example...

**Comment by andresailer** [comment]
Things are sent to the travis servers, but rejected because "abuse detected"
See: https://travis-ci.com/github/iLCSoft/MarlinReco/requests
I've written to travis support now

**Comment by andresailer** [comment]
Here is the response I received from travis support:
@MikaelBerggren, @tjunping and @yradkhorrami could you try the steps below, please

> We found that we are currently unable to detect GH token for these accounts MikaelBerggren, tjunping and yradkhorrami. Could you please follow the steps below for these users MikaelBerggren, tjunping and yradkhorrami who have the same issue that you have described? FYI, you won't need to perform this step again in the future.
> 
>     Please logout from travis-ci.com
>     Remove Travis CI from the list of Authorized OAuth Apps from the page https://github.com/settings/applications
>     Re-login to travis-ci.com and approve the permission for your accounts
>     Go to your account page https://travis-ci.com/account/repositories and Sync your account.
>

**Comment by rete** [comment]
It worked for #81 after Mikael followed the cooking recipy.
@tjunping and @yradkhorrami : Can you please follow the steps above?

**Comment by tjunping** [comment]
I've followed the cooking recipe (up to sync my account). How should I proceed? I don't see anything going on

**Comment by rete** [comment]
@tjunping I tried to re-trigger the CI, but as it was the case for @yradkhorrami it was not enough. If you log to travis-ci.com I guess your account has been flagged as "Abuse". In this case, you will see it in the top menu bar as a big red icon. If this is the case, can you please send an email to travis support explaining the situation? This solved the issue for @yradkhorrami. Once they did something to your account, please re-follow the steps above. It should end up this nightmare...

**Comment by tjunping** [comment]
@rete could you re-trigger the CI? I've contacted the Travis agent and redone the cooking recipe.

**Comment by rete** [comment]
@tjunping Looks like it's working now. Thanks a lot!

**Comment by tjunping** [comment]
what an adventure. Thanks!

File: Analysis/IsolatedLeptonTagging/include/IsolatedPhotonTaggingProcessor.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  std::string _colPhotons{};
+  std::string _isolated_photon_weights{};
+
+  bool _is_one_isophoton{};
+  float _minE{};
+
+  float _cosConeSmall{};
+  float _cosConeLarge{};  
+
+  std::vector<TMVA::Reader*> _readers{};
```
Comment (rete) [inline Analysis/IsolatedLeptonTagging/include/IsolatedPhotonTaggingProcessor.h:62]: This is actually not used. Something in mind?

Comment (tjunping) [reply Analysis/IsolatedLeptonTagging/include/IsolatedPhotonTaggingProcessor.h:62]: yes. It is reserved for MVA based method like for the isolated leptons.

**Review by rete — APPROVED** [review]

**Pull request merged**